<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>First_page</class>
 <widget class="QDialog" name="First_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Program_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>860</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>87</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87pt &quot;Bauhaus 93&quot;;
color: rgb(255, 170, 0);

</string>
   </property>
   <property name="text">
    <string>Don't Forget!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Start!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Background_image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>962</width>
     <height>542</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Background_image_qrc_file.qrc">
     <normaloff>:/BGI_sets/First_bgi.jpg</normaloff>:/BGI_sets/First_bgi.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
  </widget>
  <zorder>Background_image</zorder>
  <zorder>Program_name</zorder>
  <zorder>start_button</zorder>
 </widget>
 <resources>
  <include location="Background_image_qrc_file.qrc"/>
 </resources>
 <connections/>
</ui>
